data/src/stream.rs

```diff
@@ -75,7 +75,12 @@ pub fn run(
 ) -> impl futures::Stream<Item = Update> {
     let (sender, receiver) = mpsc::unbounded();
 
-    let runner = stream::once(_run(server, proxy, sender)).map(|_| unreachable!());
+    let runner = stream::once(async {
+        // Spawn to unblock backend from iced stream which
+        // has backpressure
+        tokio::spawn(_run(server, proxy, sender));
+        future::pending().await
```

Comment:
I believe we can simply `await` the `spawn` result here.

Reply:
Won't that tie that future back to the stream?

Reply:
We need it to stay detached

Reply:
No, the handle returned is a future that resolves once the spawned task finishes. It is not the task itself.

Reply:
Yup makes sense, I just read the docs further. Thanks!